package.json

```diff
@@ -16,6 +16,7 @@
   "repository": "expressjs/cookie-session",
   "dependencies": {
     "cookies": "0.8.0",
+    "crypto": "^1.0.1",
```

Comment:
https://www.npmjs.com/package/crypto

> This package is no longer supported and has been deprecated. To avoid malicious use, npm is hanging on to the package name.
> 
> It's now a built-in Node module. If you've depended on crypto, you should switch to the one that's built-in.